pom.xml

```diff
@@ -502,6 +508,41 @@
 				</plugins>
 			</build>
 		</profile>
+		
+		<profile>
+			<id>nonRelease</id>
+			<activation>
+				<property>
+					<name>!performRelease</name>
+				</property>
+			</activation>
+			<properties>
+				<skipNexusStagingDeployMojo>${isReleaseVersion}</skipNexusStagingDeployMojo>
+			</properties>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>build-helper-maven-plugin</artifactId>
+						<executions>
+							<execution>
+								<id>determine-build-type</id>
+								<goals>
+									<goal>regex-property</goal>
+								</goals>
+								<configuration>
+									<name>isReleaseVersion</name>
+									<value>${project.version}</value>
+									<regex>^(?!.*-SNAPSHOT$).*$</regex>
```

Comment:
What do this dark regex do?!

Reply:
ah, the old zero-length negative lookahead gambit... matches anything that doesn't end with `-SNAPSHOT`. The additional `.*$` is necessary because we need the pattern to match the entire string. I couldn't think of a better way - do you have one?

Reply:
Ah yes, I seem to recall that one. Shame we can't have a simpler `isSnapshotVersion` and negate it. Again, probably worth a comment to say what the property value is for release and snapshot versions.

Reply:
I heard you like double negatives... so I put double negatives round your double negatives. Seriously, I've really tried to make this make some sort of sense... did it work?